openspec/changes/marketplace-06-ci-module-signing/design.md

```
@@ -0,0 +1,95 @@
+## Decisions
+
+### Decision 1: Reuse `scripts/sign-modules.py` with `--changed-only` and packages root
+
+**Chosen**: Run `sign-modules.py --changed-only --base-ref "origin/<base>" --bump-version patch
+--payload-from-filesystem` from the repo root. The script's `_iter_manifests()` discovers
+manifests under `src/specfact_cli/modules/` and `modules/`; for this repo those directories are
+absent, so `--changed-only` must be combined with explicit manifest discovery or the script
+patched to also check `packages/`.
+
+**Approach**: Pass manifests explicitly by discovering them in the workflow step:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)